### ISSUE TITLE
Holes in Count Class Lookup Table

### ISSUE THREAD

**Issue by collinmay** [AFLplusplus/AFLplusplus#1083]
In 2a994e457a75c28272373ba24cd4158239c007fd, this array initializer was rewritten so as not to require special gcc syntax. However, it was rewritten incorrectly and there are two large holes in the lookup table between [64, 127] and [192, 255].

https://github.com/AFLplusplus/AFLplusplus/blob/353d402aaf9296c7dbd47e66fbbc6e59179c4e44/src/afl-fuzz-bitmap.c#L160-L176

```
[0x0000f7a0]> s obj.count_class_lookup8
[0x0004fc00]> px
- offset -   0 1  2 3  4 5  6 7  8 9  A B  C D  E F  0123456789ABCDEF
0x0004fc00  0001 0204 0808 0808 1010 1010 1010 1010  ................
0x0004fc10  2020 2020 2020 2020 2020 2020 2020 2020                  
0x0004fc20  4040 4040 4040 4040 4040 4040 4040 4040  @@@@@@@@@@@@@@@@
0x0004fc30  4040 4040 4040 4040 4040 4040 4040 4040  @@@@@@@@@@@@@@@@
0x0004fc40  0000 0000 0000 0000 0000 0000 0000 0000  ................
0x0004fc50  0000 0000 0000 0000 0000 0000 0000 0000  ................
0x0004fc60  0000 0000 0000 0000 0000 0000 0000 0000  ................
0x0004fc70  0000 0000 0000 0000 0000 0000 0000 0000  ................
0x0004fc80  8080 8080 8080 8080 8080 8080 8080 8080  ................
0x0004fc90  8080 8080 8080 8080 8080 8080 8080 8080  ................
0x0004fca0  8080 8080 8080 8080 8080 8080 8080 8080  ................
0x0004fcb0  8080 8080 8080 8080 8080 8080 8080 8080  ................
0x0004fcc0  0000 0000 0000 0000 0000 0000 0000 0000  ................
0x0004fcd0  0000 0000 0000 0000 0000 0000 0000 0000  ................
0x0004fce0  0000 0000 0000 0000 0000 0000 0000 0000  ................
0x0004fcf0  0000 0000 0000 0000 0000 0000 0000 0000  ................
```
If an edge is covered with a frequency that falls into one of these holes, it will be missed, reducing the effectiveness of new coverage detection.

**Comment by vanhauser-thc** [comment]
@collinmay you are right! thanks for reporting! weird that we do not see a regression in our regular testing. but it could have eaten up improvements we made in other parts..

@hexcoder- please note the bug

**Comment by vanhauser-thc** [comment]
fixed in d4a8a9d
